upload/library/Sidane/Threadmarks/DataWriter/Post.php

```diff
@@ -6,6 +6,11 @@ protected function _messagePostSave()
   {
     parent::_messagePostSave();
 
+    if ($this->isChanged('thread_id'))
```

Comment:
This isn't working. `$this->isChanged('thread_id')` is returning `null` so the threadmark is never updated.

Digging a bit deeper, the `_newData` property on the data writer is empty, hence why `isChanged` returns `null`:

    ["_newData":protected]=> array(0) { }